docs/samples/Microsoft.ML.Samples/Program.cs

```diff
@@ -9,6 +9,7 @@ internal static class Program
         static void Main(string[] args)
```

Comment:
>Main [](start = 20, length = 4)

who would use this app? Is it just for development testing or would the user be using it too? If so, does it make sense to change Samples to class library instead of a console app?

If we want the user to run them, we should define arguments like this:
Samples.exe Trainers.SdcaRegression
Samples.exe Transformers.ConcatEstimator

Reply:
it is just for us, to test the samples when we need to; typically when we develop them. Console app works well for testing. 

---
In reply to: [224556422](https://github.com/dotnet/machinelearning/pull/1189#discussion_r224556422) [](ancestors = 224556422)